test/aio5gc/msg/nas/codec/encoder.go

```diff
@@ -0,0 +1,71 @@
+package codec
+
+import (
+	"fmt"
+	"my5G-RANTester/test/aio5gc/context"
+
+	"github.com/free5gc/nas"
+	"github.com/free5gc/nas/security"
+	log "github.com/sirupsen/logrus"
+)
+
+func Encode(ue *context.UEContext, msg *nas.Message) ([]byte, error) {
```

Comment:
Can it be factorized with PacketRusher?

Reply:
yes, we should also be able to factorize authentication. I will do it in another pr.